### PR TITLE
Add test to increase coverage

### PR DESCRIPTION
**Pull request by JSAssassin**
Increases coverage to 93.06%

### CODE REVIEW

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

**Review by dlongley — APPROVED** [review]
Ideally this would be checked by @mattcollier to ensure the test matches expected behavior.

File: test/mocha/10-api.js [hunk 1]
```diff
@@ -44,4 +44,76 @@ describe('readiness', () => {
     result.dependencies.ready1.ready.should.equal(true);
     result.dependencies.ready2.ready.should.equal(false);
   });
+  it('should throw error if readiness protocol is not supported',
```
Comment (mattcollier) [inline test/mocha/10-api.js]: This looks like it returns an error, not throws an error.

File: test/mocha/10-api.js [hunk 2]
```diff
@@ -44,4 +44,76 @@ describe('readiness', () => {
...
+
+      should.exist(result);
+      result.valid.should.equal(false);
+      const {error: err} = result;
+      should.exist(err);
+      err.name.should.equal('DataError');
+      err.message.should.equal('Readiness dependency "ready3" is invalid.');
+      err.details.httpStatusCode.should.equal(500);
+      err.cause.message.should.equal('HTTP readiness protocol "urn:" not ' +
+      'supported. Supported protocols are: https:, http:');
```
Comment (mattcollier) [inline test/mocha/10-api.js]: indent second line when doing a string continuation.

File: test/mocha/10-api.js [hunk 3]
```diff
@@ -44,4 +44,76 @@ describe('readiness', () => {
...
+    async function() {
+      let result;
+      let err;
+      try {
+        result = await readiness.registerType(
+          {type: 'httpGet', run: async () => {}, validate: async () => {}});
+      } catch(e) {
+        err = e;
+      }
+
```
Comment (mattcollier) [inline test/mocha/10-api.js]: inconsistent use of white space.  There is white space here after try/catch but not in the test starting on L87.  Please try to be consistent throughout.

**Review by mattcollier — APPROVED** [review]
TY, style nits.

File: test/mocha/10-api.js (hunk 1 above)
Comment (JSAssassin) [reply test/mocha/10-api.js]: Thanks, updated 4b29d14

File: test/mocha/10-api.js (hunk 2 above)
Comment (JSAssassin) [reply test/mocha/10-api.js]: addressed 4b29d14

File: test/mocha/10-api.js (hunk 3 above)
Comment (JSAssassin) [reply test/mocha/10-api.js]: addressed 4b29d14

**Pull request merged**